MPU/include/driverioHW.h

```diff
@@ -0,0 +1,167 @@
+/**
+ * @file driverioHW.h
+ * @author Nick DePatie
+ * @date 2022-07-20
+ */
+
+#ifndef DRIVERIOHW_H
+#define DRIVERIOHW_H
+
+#include <nerduino.h>
+
+#define SPEAKER_DURATION        1500    //in milliseconds
+#define LED_BLINK_TIME          1000    //in milliseconds
+#define BUTTON_DEBOUNCE_TIME    50      //in milliseconds
+
+typedef enum ButtonState_t
+{
+    NOT_PRESSED,
+    DEBOUNCING,
+    PRESSED
+} ButtonState_t;
+
+
+class DASHBOARD
+{
+    public:
+        DASHBOARD();
+        ~DASHBOARD();
+};
+
+
+class BUTTON
+{
+    private:
+        ButtonState_t state = NOT_PRESSED;
+        Timer debounce;
+        uint8_t pin;
+
+        /**
+         * @brief Flag that indicates that the button has passed debounce and hasn't been read yet
+         */
+        bool readyToReadPulse = false;
+
+    public:
+        BUTTON();
```

Comment:
Probably should remove default constructor in all these HW classes.

Reply:
Done